pyogrio/tests/test_geopandas_io.py

```diff
@@ -342,12 +371,31 @@ def test_read_write_datetime_tz_with_nulls(tmp_path, use_arrow):
         {"dates": dates, "geometry": [Point(1, 1), Point(1, 1), Point(1, 1)]},
         crs="EPSG:4326",
     )
-    fpath = tmp_path / "test.gpkg"
+    fpath = tmp_path / f"test{ext}"
     write_dataframe(df, fpath, use_arrow=use_arrow)
     result = read_dataframe(fpath, use_arrow=use_arrow)
     if use_arrow:
-        # with Arrow, the datetimes are always read as UTC
-        df["dates"] = df["dates"].dt.tz_convert("UTC")
+        if ext == ".fgb":
+            # when FlatGeoBuffer is read with Arrow, for datetimes with equal timezone,
+            # a column type with the appropriate minutes offset is returned.
```

Comment:
I don't really understand this line? (or this case is not actually being tested?)